gopass: 1.14.8 -> 1.14.9

kilianar opened the pull request:
###### Description of changes
https://github.com/gopasspw/gopass/releases/tag/v1.14.9

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

kilianar committed:
gopass: 1.14.8 -> 1.14.9

https://github.com/gopasspw/gopass/releases/tag/v1.14.9

sikmir commented:
Result of `nixpkgs-review pr 193402` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>git-credential-gopass</li>
    <li>gopass</li>
    <li>gopass-hibp</li>
    <li>gopass-jsonapi</li>
    <li>gopass-summon-provider</li>
  </ul>
</details>

sikmir commented:
Could you please update git-credential-gopass, gopass-hibp, gopass-jsonapi, gopass-summon-provider to 1.14.9 as well?

kilianar committed:
git-credential-gopass: 1.14.7 -> 1.14.9

- https://github.com/gopasspw/git-credential-gopass/releases/tag/v1.14.8
- https://github.com/gopasspw/git-credential-gopass/releases/tag/v1.14.9

kilianar committed:
gopass-hibp: 1.14.7 -> 1.14.9

- https://github.com/gopasspw/gopass-hibp/releases/tag/v1.14.8
- https://github.com/gopasspw/gopass-hibp/releases/tag/v1.14.9

kilianar committed:
gopass-jsonapi: 1.14.7 -> 1.14.9

- https://github.com/gopasspw/gopass-jsonapi/releases/tag/v1.14.8
- https://github.com/gopasspw/gopass-jsonapi/releases/tag/v1.14.9

kilianar committed:
gopass-summon-provider: 1.14.7 -> 1.14.9

- https://github.com/gopasspw/gopass-summon-provider/releases/tag/v1.14.8
- https://github.com/gopasspw/gopass-summon-provider/releases/tag/v1.14.9

maxhbr reviewed (approved):
LGTM

sikmir commented:
Result of `nixpkgs-review pr 193402` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>git-credential-gopass</li>
    <li>gopass</li>
    <li>gopass-hibp</li>
    <li>gopass-jsonapi</li>
    <li>gopass-summon-provider</li>
  </ul>
</details>
